presto-spi/src/main/java/com/facebook/presto/spi/block/ArrayBlock.java

```diff
@@ -26,6 +26,7 @@
 
 public class ArrayBlock
         extends AbstractArrayBlock
+        implements Block
```

Comment:
`AbstractArrayBlock` already implements `Block` right? So this looks unnecessary?